node/packages/aws-lambda-sdk/instrument/lib/auto-send-spans.js

```diff
@@ -27,9 +26,8 @@ const sendSpans = () => {
     },
     spans: pendingSpans.map((span) => {
       const result = span.toProtobufObject();
-      for (const tagName of span.tags.keys()) {
-        if (devModeOnlyTags.has(tagName)) span.tags.delete(tagName);
-      }
+      span.input = null;
```

Comment:
Since you are returning `result` wouldnt we need to null the `result` object attributes? 🤔

Reply:
`result` is for dev mode. One what's left on `span` goes to CW logs. So this logic, retrieves full data for dev mode, and clears data we do not want to be logged in CW